app/Platform/RouteServiceProvider.php

```diff
@@ -64,6 +75,12 @@ protected function mapWebRoutes(): void
 
                 Route::get('games', [GameController::class, 'index'])->name('game.index');
 
+                Route::get('hubs', [HubController::class, 'show'])
+                    ->defaults('gameSet', GameSet::CentralHubId)
+                    ->name('hubs.index');
```

Comment:
`/hubs` is just a rewrite/fallthrough to `/hub/{gameSet}`, with the central hub ID hardcoded for its `GameSet`.